utils/events/chat_helper_ticket_sys.py

```diff
@@ -1314,6 +1314,16 @@ async def sendCHTKTView(self, interaction: discord.Interaction):
             > {Emoji.ss_arrow} Send a picture of your assignment title in your direct messages as per the bot instructions.""",
             view=TicketButton(self.bot),
         )
+    
+    @app_commands.command(
+        name="title", description="Prompts for assignment title."
+    )
+    @app_commands.guilds(MainID.g_main)
+    @slash_is_bot_admin()
+    async def sendTitle(self, interaction: discord.Interaction):
```

Comment:
Nothing wrong here, although could you use camel casing for this cmd name?
eg: `async def send_title(self, interaction: discord.Interaction):`

Reply:
will do.